.dependency-cruiser.js

```diff
@@ -183,7 +238,140 @@ module.exports = {
           'npm-peer'
         ]
       }
-    }
+    },
+
+    /* application-specific architecture rules */
+    {
+      name: 'dependencies-of-shared',
+      severity: 'error',
+      from: {
+        path: dir('src/shared')
+      },
+      to: {
+        pathNot: [
+          dir('src/shared'),
+          dir('src/fs'),
```

Comment:
This is only used in test code, and should be forbidden otherwise.

Reply:
I've removed `src/fs` from the allowed import paths and adapted the rule to only apply to production (non-test) code.